src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/FormLoginFactory.php

```diff
@@ -62,7 +67,7 @@ public function addConfiguration(NodeDefinition $node)
 
                     return $v;
                 })
-                ->end()
+            ->end()
```

Comment:
I guess the real deprecaction should be added here also, like https://github.com/bjcullinan/symfony/blob/dbe32eabd5553f9c6df3a2d9909303e1d77510f1/src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php#L253